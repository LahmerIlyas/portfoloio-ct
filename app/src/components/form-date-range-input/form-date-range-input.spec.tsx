import React from 'react';
import renderer from 'react-test-renderer';
import { Formik } from 'formik';
import { FormDateRangeInput } from '.';

describe('<FormDateRangeInput />', () => {
  it('renders correctly', async () => {
    const tree = renderer
      .create(
        <Formik initialValues={{}} onSubmit={() => {}}>
          <FormDateRangeInput />
        </Formik>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
