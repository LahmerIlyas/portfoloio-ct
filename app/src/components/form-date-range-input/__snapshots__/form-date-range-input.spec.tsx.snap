// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormDateRangeInput /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Signika-SemiBold",
      }
    }
  >
    From
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "gray",
        "borderRadius": 8,
        "flexGrow": 1,
        "marginHorizontal": 4,
        "opacity": 0.5,
        "paddingHorizontal": 8,
        "paddingVertical": 16,
      }
    }
  >
    <Text>
      
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontFamily": "Signika-SemiBold",
      }
    }
  >
    To
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "gray",
        "borderRadius": 8,
        "flexGrow": 1,
        "marginHorizontal": 4,
        "opacity": 0.5,
        "paddingHorizontal": 8,
        "paddingVertical": 16,
      }
    }
  >
    <Text>
      
    </Text>
  </View>
</View>
`;
